### PR TITLE
Scale coords update

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Adds new functionality to scale_coords, extending the range of formats allowed for velocity and testing the output messages upon failure to ensure the correct failure is triggered.

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- Run with new velocity input options:
```
from pysat.utils import coords
out1 = coords.scale_units('m s$^{-1}$', 'm/s')
out2 = coords.scale_units('m/s', 'm s$^{-1}$')
out3 = coords.scale_units('m/s', 'km s$^{-1}$')
out4 = coords.scale_units('cm  s$^{-1}$', 'km\s')
print(out1, out2, out3, out4)
1.0, 1.0, 1000.0, 100000.0
```

- Trigger new warning message that tells you both input and output are unknown
```
coords.scale_units('climb', 'rocks')
```
Returns:  `ValueError: Cannot scale rocks and climb, unknown units`

**Test Configuration**:
* OS X Sierra

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``master``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
...
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.X.X] - 2019-07-17
+- New Features
+   - Added new velocity format options to utils.coords.scale_units
+   - Improved failure messages for utils.coords.scale_units
+- Bug fix
+   - Fixed implementation of utils routines in model_utils and jro_isr
+
 ## [2.0.0] - 2019-07-??
```
Comment (jklenzing) [inline CHANGELOG.md]: We should update with the actual release date of 2.0.0

Comment (rstoneback) [reply CHANGELOG.md]: My bad, sorry about that.

Comment (jklenzing) [reply CHANGELOG.md]: fixed.

**Comment by jklenzing** [comment]
Oops.  Meant to add the model_utils tests to #257.

File: pysat/tests/test_utils_coords.py [hunk 2]
```diff
@@ -150,22 +151,62 @@ def test_scale_units_vel(self):
...
 
-    def test_scale_units_bad(self):
-        """Test scale_units for mismatched input"""
+    def test_scale_units_bad_output(self):
+        """Test scale_units for unknown output unit"""
 
         assert_raises(ValueError, coords.scale_units, "happy", "m")
+        try:
+            coords.scale_units('happy', 'm')
+            verr = None
```
Comment (jklenzing) [inline pysat/tests/test_utils_coords.py]: Why is the `verr` statement here?  According to coveralls, it is not run.

**Comment by aburrell** [comment]
@jklenzing it looks like the routine I am changing is completely covered, and test_utils_coords.py is now at 100%, so I think the issue you raised is resolved.  Requesting approval to merge and move on with our lives.

**Pull request merged**